src/plugins/mer/meroptionswidget.cpp

```diff
@@ -135,14 +135,15 @@ QString MerOptionsWidget::searchKeyWordMatchString() const
 
 void MerOptionsWidget::store()
 {
-    QProgressDialog progress(this);
+    QProgressDialog progress(ICore::mainWindow());
     progress.setWindowModality(Qt::WindowModal);
     QPushButton *cancelButton = new QPushButton(tr("Abort"), &progress);
     cancelButton->setDisabled(true);
     progress.setCancelButton(cancelButton);
     progress.setMinimumDuration(2000);
     progress.setMinimum(0);
     progress.setMaximum(0);
+    progress.show();
```

Comment:
It is not desired to call show here. Instead, setting minimum duration orders for showing the dialog only if the operation takes more time than anticipated.